app/views/pages/index.html.erb

```diff
@@ -134,14 +134,15 @@
       <article class="HomepageCategory">
         <%= link_to "/docs/agent/v3/elastic-ci-aws/elastic-ci-stack-overview", class: "HomepageCategory__header" do %>
           <h3 class="HomepageCategory__heading">Elastic CI Stack</h3>
-          <p class="HomepageCategory__subtext">Get started with Buildkite Elastic CI Stack for:</p>
+          <p class="HomepageCategory__subtext">Set up your Buildkite Elastic CI Stack</p>
           <div class="HomepageCategory__icon" aria-hidden>
             <%= image_tag "icons/elastic.svg", alt: "Icon: Elastic CI Stack", class: "HomepageCategory__icon-img" %>
           </div>
         <% end %>
         <ul class="HomepageCategory__list">
           <li class="HomepageCategory__list-item"><%= link_to "AWS Linux and Windows", "/docs/agent/v3/elastic-ci-aws", class: 'HomepageCategory__list-item-link' %></li>
           <li class="HomepageCategory__list-item"><%= link_to "AWS EC2 for Mac", "/docs/agent/v3/elastic-ci-stack-for-ec2-mac/autoscaling-mac-metal", class: 'HomepageCategory__list-item-link' %></li>
+          <li class="HomepageCategory__list-item"><%= link_to "Customize instances with bootstrap script", "/docs/agent/v3/elastic-ci-aws#customizing-instances-with-a-bootstrap-script", class: 'HomepageCategory__list-item-link' %></li>
```

Comment:
style: I suggest parallelism with both the preceding items (noun, noun, noun instead of noun, noun, imperative) and the section heading on the page you end up on. Makes the whole thing feel a bit more cohesive.

```suggestion
          <li class="HomepageCategory__list-item"><%= link_to "Customizing instances with a bootstrap script", "/docs/agent/v3/elastic-ci-aws#customizing-instances-with-a-bootstrap-script", class: 'HomepageCategory__list-item-link' %></li>
```

Reply:
Don't get me wrong, I hate gerunds (I'd go with imperatives _everywhere_ given the choice, for a host or reasons). But looking across the content of each of the boxes on this page, -ing is the most common thing (e.g., take a look at the bodies of the "Configuring pipelines" and "Running Buildkite agents" boxes).

Reply:
Just pushed up 7f4af0e1e3c6a3e680de11242264326bf911233c for Option 4.